rllib/policy/policy.py

```diff
@@ -102,21 +102,42 @@ def __eq__(self, other: "PolicySpec"):
         )
 
     def serialize(self) -> Dict:
+        from ray.rllib.algorithms.registry import get_policy_class_name
```

Comment:
Add the import to the top of the file

Reply:
I can't 😭 
circular dependency somehow ...